src/Altinn.App.Core/Features/Signing/SigningService.cs

```diff
@@ -196,6 +201,33 @@ AltinnSignatureConfiguration signatureConfiguration
         return signeeContexts;
     }
 
+    // <inheritdoc />
+    public async Task<List<OrganisationSignee>> GetAuthorizedOrganisations(
```

Comment:
Siden det er i kontekst av signatarer, så forslag å reflektere det i navnet?
```suggestion
    public async Task<List<OrganisationSignee>> GetAuthorizedOrganisationSignees(
```

Reply:
🙌